Remotion/ObjectBinding/Web.IntegrationTests/App.config

```diff
@@ -25,6 +23,7 @@
       screenshotDirectory=".\WebTestingOutput"
       logsDirectory=".\WebTestingOutput"
       closeBrowserWindowsOnSetUpAndTearDown="false"
+      cleanUpUnmatchedDownloadedFiles="false"
```

Comment:
Why the change?

Reply:
There is no change in configuration its just an explicit default value to make patching the file easier.